core/src/test/java/apoc/refactor/GraphRefactoringTest.java

```diff
@@ -333,27 +330,31 @@ public void deleteAndReconnectWithMergeRelConfigAndPropertiesCombine() {
                 db,
                 "MATCH p=(f:One)-->(b:Two)-->(c:Three)-->(d:Four)-->(e:Five) WITH p, [b,d] as list CALL apoc.refactor.deleteAndReconnect(p, list, {properties: 'combine', relationshipSelectionStrategy: 'merge'}) YIELD nodes, relationships RETURN nodes, relationships",
                 (row) -> {
-                    List<Node> nodes = (List<Node>) row.get("nodes");
-                    assertEquals(3, nodes.size());
-                    Node node1 = nodes.get(0);
-                    assertEquals(singletonList(label("One")), node1.getLabels());
-                    Node node2 = nodes.get(1);
-                    assertEquals(singletonList(label("Three")), node2.getLabels());
-                    Node node3 = nodes.get(2);
-                    assertEquals(singletonList(label("Five")), node3.getLabels());
-                    List<Relationship> rels = (List<Relationship>) row.get("relationships");
-                    assertEquals(2, rels.size());
-                    Relationship rel1 = rels.get(0);
-                    assertEquals("ALPHA_BETA", rel1.getType().name());
-                    assertEquals("f", rel1.getProperty("e"));
-                    assertThat(Arrays.asList((String[]) rel1.getProperty("a")), containsInAnyOrder("b", "d"));
-                    assertEquals("h", rel1.getProperty("g"));
-                    Relationship rel2 = rels.get(1);
-                    assertEquals("GAMMA_DELTA", rel2.getType().name());
-                    assertThat(Arrays.asList((String[]) rel2.getProperty("aa")), containsInAnyOrder("one", "bb"));
-                    assertEquals("ff", rel2.getProperty("ee"));
-                    assertEquals("dd", rel2.getProperty("cc"));
-                    assertNotNull(row.get("nodes"));
+                    assertThat(row.get("nodes"))
```

Comment:
I had some test failures of the unhelpful kind "false was not equal to true". So refactored some tests to help me out unserstanding that.